### PR TITLE
Time out bar

### PR DESCRIPTION
**Pull request by HelloUniverseNo31415**
# Checklist:

Related work items: #118 

Preview screenshot: 
image

- [ ] Tests have been added for new functionality
- [ ] Work has been reviewed by other team members
- [ ] Work has been reviewed by @craigayre or @eclennett

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -61,4 +61,4 @@ typings/
 .next
 
 # SASS generated css files
-src/**/*.css
+src/**/*.css
```
Comment (cainy393) [inline .gitignore]: Can we add the new line back in

File: src/components/TimeoutNotification/TimeoutNotification.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import React, {Component} from 'react';
+import {Redirect} from 'react-router';
+import './TimeoutNotification.css';
+
+const WAIT_BEFORE_DISPLAY = 10;
```
Comment (cainy393) [inline src/components/TimeoutNotification/TimeoutNotification.js]: 10 seconds seems a bit short

Comment (HelloUniverseNo31415) [reply src/components/TimeoutNotification/TimeoutNotification.js]: This is for no user interaction at all. Also it won't be on the item scan page as that has no user to screen interaction, so it should be long enough

File: src/components/TimeoutNotification/TimeoutNotification.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+	dismissMessage = () => {
+		clearInterval(this.interval);
+		this.resetTimer();
+		this.setState({displayWarning: false, countdown: COUNTDOWN});
+	};
+
+	render() {
+		if (this.state.countdown === 0) {
+			clearInterval(this.interval);
```
Comment (craigayre) [inline src/components/TimeoutNotification/TimeoutNotification.js]: Could this moved to keep render pure of side effects? We could do this in `countdownTick` or `componentDidUpdate`

e.g.

```
componentDidUpdate(props, state) {
  if (state.countdown === 0) {
    clearInterval(this.interval)
  }
}
```

**Pull request merged**